objects/src/notes/script.rs

```diff
@@ -1,13 +1,11 @@
-use super::{
-    Assembler, AssemblyContext, AssemblyContextType, CodeBlock, Digest, NoteError, ProgramAst,
-};
+use super::{Assembler, AssemblyContext, CodeBlock, Digest, NoteError, ProgramAst};
 
 #[derive(Debug, Clone)]
 #[cfg_attr(feature = "serde", derive(serde::Deserialize, serde::Serialize))]
 pub struct NoteScript {
-    hash: Digest,
+    pub hash: Digest,
     #[cfg_attr(feature = "serde", serde(with = "serialization"))]
-    code: ProgramAst,
+    pub code: ProgramAst,
 }
```

Comment:
I don't think we need to make these public - we already have accessors for these fields (i.e., `hash()` and `code()`).